### PR TITLE
#4201 list components

### PR DESCRIPTION
**Pull request by desfero**
**TODO:**
- [x] Get a proper icon for scan QR code button from @andywalton 

**Technical notes**
- react-navigation was updated. The original reason for the update was to see if it fixes the tab navigation padding-top jumping while changing tabs but it didn't fix a bug. The bug was fixed by applying `forceInset={ top: "always" }` to react-native-community/safe-view.

**Testing notes**
This PR adds an `Asset` UI component and also connects that component to Home/Portfolio/Wallet views.

Android: https://i.diawi.com/NiH3z6
IOs: https://i.diawi.com/gRjX1Z

- [x] Asset component is properly implemented as in zeplin
- [x] Asset component is properly plugged into home view 
- [x] Home view contains a proper header from zeplin
- [x] Wallet connect and logout actions were moved to `Profile` (for now they are not styled properly)
- [x] Asset component is properly plugged into Portfolio and Wallet view
- [x] A proper header implementation is used on Portfolio and Wallet view
- [x] A header animation on Portfolio and Wallet is good enough for the user

### CODE REVIEW

File: packages/wallet/app/components/home/HomeScreen.tsx [hunk 1]
```diff
@@ -1,79 +1,87 @@
...
-    walletConnectPeer: walletConnectModuleApi.selectors.selectWalletConnectPeer(state),
-    user: nonNullable(authModuleAPI.selectors.selectUser(state)),
-  }),
-  dispatchToProps: dispatch => ({
-    walletConnectDisconnect: (peerId: string) =>
-      dispatch(walletConnectModuleApi.actions.disconnectFromPeer(peerId)),
-    logout: () => dispatch(authModuleAPI.actions.logout()),
-  }),
+const HomeScreen = appConnect<TStateProps>({
+  stateToProps: () => ({}),
```
Comment (desfero) [inline packages/wallet/app/components/home/HomeScreen.tsx:84]: gonna keep it as empty as soon we will feed proper data into the view

File: packages/wallet/app/components/profile/ProfileScreen.tsx [hunk 2]
```diff
@@ -43,15 +56,34 @@ const ProfileLayout: React.FunctionComponent<TStateProps> = ({ authWallet }) =>
...
       <View style={styles.menuHeader}>
         <Avatar name={authWallet.name} />
 
         <AddressShare address={authWallet.address} style={styles.addressShare} />
       </View>
 
       <Menu items={items} />
-    </Screen>
+
+      <Text>Wallet connect peer: </Text>
```
Comment (desfero) [inline packages/wallet/app/components/profile/ProfileScreen.tsx:68]: wallet connect and logout gonna be styled while working on #4217

File: packages/wallet/app/components/shared/buttons/ButtonIcon.tsx [hunk 3]
```diff
@@ -67,8 +67,8 @@ const styles = StyleSheet.create({
     justifyContent: "center",
     paddingTop: 12,
     paddingBottom: 10,
-    paddingLeft: 10,
-    paddingRight: 10,
+    paddingLeft: 7,
```
Comment (desfero) [inline packages/wallet/app/components/shared/buttons/ButtonIcon.tsx:70]: Given that we manually add empty space to align height we need to reduce horizontal padding manually to align with a design-system button icon width

File: packages/wallet/app/components/home/HomeScreen.tsx [hunk 4]
```diff
@@ -1,79 +1,87 @@
...
-            layout={EButtonLayout.PRIMARY}
-            onPress={() => walletConnectDisconnect(walletConnectPeer.id)}
-          >
-            Disconnect
-          </Button>
-        </>
-      ) : (
-        <Text>not connected</Text>
-      )}
+const styles = StyleSheet.create({
```
Comment (arjunu) [inline packages/wallet/app/components/home/HomeScreen.tsx:70]: There are no spaces between the items in the design. And also the border radius is 12.
image

File: packages/wallet/app/components/portfolio/PortfolioScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import { toEquityTokenSymbol } from "@neufund/shared-utils";
+import React from "react";
+import { StyleSheet } from "react-native";
+
+import { spacingStyles } from "../../styles/spacings";
+import { Asset, EAssetType } from "../shared/asset/Asset";
+import { HeaderScreen } from "../shared/HeaderScreen";
+
+const PortfolioScreen: React.FunctionComponent = () => (
```
Comment (arjunu) [inline packages/wallet/app/components/portfolio/PortfolioScreen.tsx:9]: this page doesn't scroll

**Comment by arjunu** [comment]
Not part of this PR, but FYI. Ellipsize the name: 
image

File: packages/wallet/app/components/home/HomeScreen.tsx (hunk 4 above)
Comment (desfero) [reply packages/wallet/app/components/home/HomeScreen.tsx:70]: https://github.com/neufund/platform-frontend/issues/4201 we decide for now to reuse the same components we have in wallet and portfolio views

**Pull request merged**